larpandora/LArPandoraInterface/LArPandoraGeometry.cxx

```diff
@@ -389,19 +426,29 @@ namespace lar_pandora {
           if (theTpc1.DriftDirection() != theTpc2.DriftDirection()) continue;
 
           // ATTN: In dual phase scenario propagate the W->U and Y->V mapping as described above.
+          //REPLACEMENT
+          const float dThetaU(detType->WireAngleU(itpc1, icstat) - detType->WireAngleU(itpc2, icstat));
+          const float dThetaV(detType->WireAngleV(itpc1, icstat) - detType->WireAngleV(itpc2, icstat));
+          const float dThetaW(detType->WireAngleW(itpc1, icstat) - detType->WireAngleW(itpc2, icstat));
+          /*
+          bool useYPlane(false);
+          bool isDualPhase(false);
+          int nWirePlanes=3;
           const geo::View_t pandoraUView(isDualPhase ? geo::kW : geo::kU);
           const geo::View_t pandoraVView(isDualPhase ? geo::kY : geo::kV);
-          const float dThetaU(theGeometry->WireAngleToVertical(pandoraUView, itpc1, icstat) -
+          const float dThetaU2(theGeometry->WireAngleToVertical(pandoraUView, itpc1, icstat) -
```

Comment:
Is this just a sanity check for the angle to vertical, as it doesn't look like it's used beyond the cout?

Reply:
oh this was supposed to be commented out.  It will eventually be removed (it was a debugging tool).

Reply:
This was commented out and will be removed during cleanup